test/bridge.js

```diff
@@ -19,10 +24,16 @@ const should = chai
   .use(chaiBigNumber(web3.BigNumber))
   .should();
 
+const sendTransaction = (target, method, args, values, opts) => {
```

Comment:
How does this compare to this: https://github.com/leapdao/leap-contracts/commit/09a42057fa1c625abe59e3799d870ed9800a7fbe#diff-8a2e77ddc9039cfef63f7c98322c3a02R33

Reply:
This sendTransaction thing with encodeCall is needed because truffle cannot handle function calls when there are more than one function with the same name (when inheriting), in this case it is `initialize` function